`data-plane-server` launcher is broken

# Bug Report

## Describe the Bug
Compilation complains that no artifact named `org.eclipse.dataspaceconnector:common:0.0.1-SNAPSHOT` can be found when trying to build the `data-plane-launcher runtime`

### Expected Behavior
Compilation of the `data-plane-server` should be successful.

### Observed Behavior
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':launchers:dataplane:shadowJar'.
> Could not resolve all dependencies for configuration ':launchers:dataplane:runtimeClasspath'.
   > Could not find org.eclipse.dataspaceconnector:common:0.0.1-SNAPSHOT.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/maven-metadata.xml
       - https://repo.maven.apache.org/maven2/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/common-0.0.1-SNAPSHOT.pom
       - file:/C:/Users/bscholtes/.m2/repository/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/maven-metadata.xml
       - file:/C:/Users/bscholtes/.m2/repository/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/common-0.0.1-SNAPSHOT.pom
       - https://maven.iais.fraunhofer.de/artifactory/eis-ids-public/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/maven-metadata.xml
       - https://maven.iais.fraunhofer.de/artifactory/eis-ids-public/org/eclipse/dataspaceconnector/common/0.0.1-SNAPSHOT/common-0.0.1-SNAPSHOT.pom
     Required by:
         project :launchers:dataplane > org.eclipse.dataspaceconnector:data-plane-server:0.0.1-SNAPSHOT > org.eclipse.dataspaceconnector:data-plane-azure-storage:0.0.1-SNAPSHOT
```

## Steps to Reproduce
Create a runtime containing the `data-plane-server` and run `./gradlew build`

## Possible Implementation
`data-plane-azure-common` should publish an artifact.

--- a/extensions/azure/data-plane/common/build.gradle.kts
+++ b/extensions/azure/data-plane/common/build.gradle.kts
@@ -20,3 +20,12 @@ dependencies {
     api(project(":extensions:data-plane:data-plane-spi"))
     implementation(project(":common:util"))
 }
+
+publishing {
+    publications {
+        create<MavenPublication>("data-plane-azure-common") {
+            artifactId = "data-plane-azure-common"
+            from(components["java"])
+        }
+    }
+}

